Gemfile

```diff
@@ -20,6 +20,7 @@ gem 'devise'
 gem 'foundation-rails'
 gem "pry"
 gem 'textacular', '~> 3.0'
+gem 'mailcatcher'
```

Comment:
I believe the mailcatcher documentation states that you shouldn't add it to the Gemfile.

http://mailcatcher.me/
> Please don't put mailcatcher into your Gemfile. It will conflict with your applications gems at some point.

> Instead, pop a note in your README stating you use mailcatcher. Simply run gem install mailcatcher then mailcatcher to get started.